util-core/src/main/scala/com/twitter/io/ByteReader.scala

```diff
@@ -384,6 +398,41 @@ private class ByteReaderImpl(buf: Buf) extends ByteReader {
     ret
   }
 
+  private val unsignedLongMaxValue: BigInt = BigInt("18446744073709551615")
+
+  def readUnsignedLongBE(): BigInt = {
```

Comment:
> It is not necessary in this case.
Should remove that last & operation if it is unnecessary?

`BigInt(1,` Array(buf.get(pos), buf.get(pos+1), buf.get(pos+2)... )
I know that too but just do not want to allocate another Array.